e2e-tests/scheduled-backup/run

```diff
@@ -113,6 +126,27 @@ main() {
     wait_backup "minio"
     wait_backup "gcp-cs"
 
+    if [[ $(kubectl_bin get pods -o custom-columns=NAME:.metadata.name,NODE:.spec.nodeName --no-headers=true \
+            | grep ${cluster}-xb-cron-${cluster} \
+            | awk '{print $2}' \
+            | sort \
+            | uniq \
+            | wc -l \
+            | awk '{$1=$1};1') != 1 ]]; then
+        echo "Taint/toleration rules have been violated."
+        exit 1
+    fi
+
+    kubectl_bin taint nodes "${TAINTED_NODE}" backupWorker=False:NoSchedule-
+
+    if [[ $(kubectl_bin get pods -o jsonpath='{range .items[*]}{.metadata.name}:{.metadata.annotations.testName}{"\n"}' \
+            | grep ':scheduled-backup$' \
+            | wc -l ) == 0 ]]; then
+
+        echo "No backup worker pods with target annotation"
+        exit 1
+    fi
+
```

Comment:
could we also add `compare_kubectl` for backup and recovery jobs, please?
goal: I want to be sure that all fields passed into Kubernetes Job structure correctly